apps/src/applab/designElements/textarea.jsx

```diff
@@ -16,10 +16,29 @@ var TextAreaProperties = React.createClass({
     handleChange: React.PropTypes.func.isRequired,
     onDepthChange: React.PropTypes.func.isRequired
   },
-
+  componentWillMount: function () {
+    $(this.props.element).on("resizing", (function () {
+      this.setState({
+        'style-width': parseInt(this.props.element.style.width, 10),
+      });
+    }).bind(this));
+  },
+  componentWillUnmount: function () {
+    $(this.props.element).off("resizing");
+  },
+  getInitialState: function () {
+    return {
+      'style-width': parseInt(this.props.element.style.width),
```

Comment:
Our linter might complain that we're not specifying a radix for parseInt....ie. parseInt(number, 10);